src/pages/home/report/ReportActionCompose.js

```diff
@@ -590,7 +594,7 @@ class ReportActionCompose extends React.Component {
                                     )}
                                 </AttachmentPicker>
                                 <TextInputFocusable
-                                    autoFocus={this.shouldFocusInputOnScreenFocus}
+                                    autoFocus={this.shouldFocusInputOnScreenFocus || _.size(this.props.reportActions) === 1}
```

Comment:
Well, there is an issue that is preventing the keyboard to open in android but this works. So we can keep that in a separate issue.